src/calculateRentalCost.js

```diff
@@ -28,7 +28,7 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return days * 31;
```

Comment:
The rental cost calculation is not correct. Follow the task guidelines to implement the correct calculation based on the number of rental days.